Makefile

```diff
@@ -226,62 +228,62 @@ docker-ci-benchmark-team-city:
 
 .PHONY: docker-build-collection
 docker-build-collection:
-	docker build -f cmd/Dockerfile  --build-arg TARGET=collection --target production \
+	docker build -f cmd/Dockerfile  --build-arg TARGET=collection --build-arg COMMIT=$(COMMIT)  --build-arg VERSION=$(IMAGE_TAG) --target production \
```

Comment:
The software version is passed in as linker params (ld flags). The `Image_Tag` will be set to something like - `v0.16.0-patch1`. This will be the software version reported by the Ping metric.